### ISSUE TITLE
[Bug] Result pages refresh (on socket events?) states

### ISSUE THREAD

**Issue by frankcorneliusmartin** [vantage6/vantage6#1337]
**Describe the bug**
When viewing a result in the result page just after a computation has been completed you get some unexpected behavior when you unfold the run section or when you select an alternative visualization.

1. When unfolding the a run, it collapses again
2. When selecting an alternative visualization is jumps back to the initial one (but leaves the drop down selected item)

Typically it takes about 10 seconds after the results are shown before this behavior does no longer show.

**To Reproduce**
1. Create a task with two visualization options (e.g. basic OMOP queries, with function `person_count`)
3. Wait till it complets
4. Immediately select an alternative visualization or unfold a run 
5. See error

**Expected behavior**
Do not change the state when a socket event occurs?

**Comment by bartvanb** [comment]
The issue switching between different visualizations has been solved in https://github.com/vantage6/vantage6/issues/1511

The issue that unfolded `runs` refold persists, I'll fix that

**Comment by frankcorneliusmartin** [comment]
OK by me. @bartvanb we keep this one open then?

**Comment by bartvanb** [comment]
Yes, but I'll change the milestone to somewhere further in the future
